modules/upgrade/pages/client-x.adoc

```diff
@@ -1,30 +1,13 @@
 [[client-migration]]
-= Client Migration
-
-////
-Upgrading traditional clients from earlier SLE{nbsp}12 Service Packs to later SLE{nbsp}12 Service Packs is fully automated.
-For example, upgrading from SLE{nbsp}12{nbsp}SP1 to SLE{nbsp}12{nbsp}SP4.
-This also applies for other supported products based on SLE{nbsp}12.
-
-////
-
-Upgrading traditional clients from SLE{nbsp}12 with the latest Service Pack to SLE{nbsp}15 is automated, but you will need to do some preparation steps before you can upgrade.
-
-Upgrading traditional clients from earlier SLE{nbsp}15 Service Packs to later SLE{nbsp}15 Service Packs is fully automated.
-
-For supported {sle}{nbsp}15 upgrade paths, see https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-upgrade-paths.html
-
-
-
-== Upgrade Clients from SLE{nbsp}12 to SLE{nbsp}15
+= Client - X Upgrade
```

Comment:
In the nav, we have "Major Version" instead of "X Upgrade".  This also applies for all the other sections.

Reply:
Yeah, and it's "X Upgrade" in the PDF. I'll update the nav to read "X Upgrade", etc.